croud/consumers/commands.py

```diff
@@ -24,6 +24,42 @@
 from croud.util import clean_dict
 
 
+def consumers_deploy(args: Namespace) -> None:
+    body = dedent(
+        """
+    query deployConsumer($input: DeployConsumerInput) {
+        deployConsumer(input: $input) {
+            id
+            name
+        }
+    }
+    """  # noqa
+    ).strip()
+
+    vars = clean_dict(
+        {
+            "input": {
+                "name": args.consumer_name,
+                "productName": args.product_name,
+                "productTier": args.tier,
+                "tableName": args.consumer_table,
+                "tableSchema": args.consumer_schema,
+                "clusterId": args.cluster_id,
+                "projectId": args.project_id,
+                "instances": args.num_instances,
+                "eventhub_connection_string": args.eventhub_dsn,
```

Comment:
```suggestion
                "eventhubConnectionString": args.eventhub_dsn,
```
smallCamelCase instead of snake_case, please. Also in the lines below